### ISSUE TITLE
Validate existing schema by trying to use examples

### ISSUE THREAD

**Issue by jordan-evens** [CWFMF/wx_specification#8]
Please try to parse/manipulate/save or whatever the example data that is located in the [./examples](https://github.com/CWFMF/wx_specification/tree/main/examples) subdirectory to see if it works for you.

There's some basic python processing scripts in the [./example_usage](https://github.com/CWFMF/wx_specification/tree/main/example_usage) folder that might be of use for interpreting things.

There is a commeted version of [./examples/example_wx_uncommented.geojson](https://raw.githubusercontent.com/CWFMF/wx_specification/main/examples/example_wx_uncommented.geojson) located at [./examples/example_wx.geojson](https://github.com/CWFMF/wx_specification/blob/main/examples/example_wx.geojson) that has notes about the schema layout that may help.

**Comment by spydmobile** [comment]
@jordan-evens ok, if i write up some javascript/typescript code do you want me to add it to e example_usage folder? also, @SIBeckers seemed to think there is only one repo, this makes two ;-)

**Comment by SIBeckers** [comment]
Yeah I know about this one.

**Comment by SIBeckers** [comment]
> @jordan-evens ok, if i write up some javascript/typescript code do you want me to add it to e example_usage folder? also, @SIBeckers seemed to think there is only one repo, this makes two ;-)

I vote yes

**Comment by jordan-evens** [comment]
@spydmobile Yeah for sure - would be good to make sure that it's working in multiple implementations as we tweak it. If you can get a base version going I should be able to update it as we modify the spec
